tests/sqllogictests/suites/mode/standalone/explain/explain_analyze_graphical.test

```diff
@@ -0,0 +1,20 @@
+statement ok
+drop database if exists testdb
+
+statement ok
+create database testdb
+
+statement ok
+use testdb
+
+statement ok
+create table t(a int, b int)
+
+statement ok
+insert into t select number, number + 1 from numbers(1000)
+
+query T
+explain analyze graphical select * from t where a = 1
```

Comment:
Can we add some json struct validation for the result set?

Reply:
Absolutely! The `query_id` will be returned, but since it changes each time, could you guide me on how to write the SQL to account for this variability? Is it possible to use a regex like "query_id": "[a-f0-9\-]{36}" to match the dynamic query_id?

Reply:
We can write shell test scripts to verify using regex, sed, or jq. example: `https://github.com/databendlabs/databend/blob/main/tests/suites/1_stateful/02_query/02_0001_create_table_with_external_location.sh#L26`

Reply:
Hi @zhang2014 , I wrote a new test, but I encountered a permission denied error when trying to run it: 
```
    /bin/sh: 1: /runner/_work/databend/databend/tests/suites/1_stateful/02_query/02_0009_explain_profile.py: Permission denied
```
Can you tell me on how to solve this problem?

Reply:
chmod +x /runner/_work/databend/databend/tests/suites/1_stateful/02_query/02_0009_explain_profile.py

Reply:
https://github.com/databendlabs/databend/actions/runs/11387597660/job/31683755850?pr=16543 
![img_v3_02fo_857ba555-674f-4f17-815a-d6867889c4cg](https://github.com/user-attachments/assets/93c82673-c4a8-456c-b77b-9e0cc6cd06eb)
@zhang2014 I added a new test case, but it’s throwing a "Permission denied" error. I’m not sure where I need to set the appropriate permissions.

Reply:
You need to chmod this file in your local repo and commit push it to github.

we should 
```
❯  ll tests/suites/1_stateful/02_query/*.sh
-rwxr-xr-x 1 sundy sundy 1677 Jul  2 17:23 tests/suites/1_stateful/02_query/02_0001_create_table_with_external_location.sh
-rwxr-xr-x 1 sundy sundy  504 Jul  2 17:23 tests/suites/1_stateful/02_query/02_0003_create_table_with_block_per_segment.sh
-rwxr-xr-x 1 sundy sundy  641 Oct 10 13:47 tests/suites/1_stateful/02_query/02_0007_get_profile.sh
```

we should make sure `sh` files include permission with execution.